internal/saucecloud/cloud.go

```diff
@@ -200,6 +208,26 @@ func (r *CloudRunner) runJob(opts job.StartOptions) (j job.Job, interrupted bool
 	return j, false, nil
 }
 
+func (r *CloudRunner) dryRunForMobile(opts job.StartOptions) error {
+	log.Warn().Msg("Running tests in dry run mode.")
+	tmpDir, err := os.MkdirTemp("./", "sauce-app-payload-*")
+	if err != nil {
+		return err
+	}
+	log.Info().Msgf("The following test suites would have run: [%s].", opts.DisplayName)
```

Comment:
Does `opts.DisplayName` contain the list of suites to be run?

At a higher level, I'm not sure if the output of this function is sufficient. What info would be helpful for the user running in `dry-run` mode? It would probably be useful to see the suites + the configured devices. Especially so since we do the whole platform multiplication thing for mobile.

Reply:
`opts.DisplayName` is only suite name, e.g. 
```
14:18:22 INF The following test suites would have run: [saucy barista].
```
It's a good idea to append deviceName as well. Let me add it.

Reply:
I've changed the output to
```
11:09:48 INF The saucy barista test would run on Android GoogleApi Emulator Android 11.0.
11:09:48 INF The saucy barista test would run on Android GoogleApi Emulator Android 10.0.
11:09:48 INF The saucy barista test would run on Google Pixel C GoogleAPI Emulator Android 8.1.
```